### PR TITLE
Remove babel

### PR DESCRIPTION
**Pull request by cristianooliveira**
## Description

Remove babel and update minimum node version allowed to 8.
Add `yarn.lock`.

## Motivation

The initial motivation for this PR was to remove security vulnerabilities related with old packages.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -38,10 +39,16 @@ export function isValid(value) {
...
     throw new Error('Invalid Social Security Number');
   }
 
   return `${value.substr(0, value.length - 4).replace(/[\w]/g, 'X')}${value.substr(-4)}`;
 }
+
+/**
+ * Export `isvalid` and `mask`.
+ */
+module.exports = { isValid, mask };
```
Comment (rplopes) [inline src/index.js:54]: Newline above, not below 😛

File: src/index.js [hunk 2]
```diff
@@ -38,10 +39,16 @@ export function isValid(value) {
...
+function mask(value) {
   if (!isValid(value)) {
     throw new Error('Invalid Social Security Number');
   }
 
   return `${value.substr(0, value.length - 4).replace(/[\w]/g, 'X')}${value.substr(-4)}`;
 }
+
+/**
+ * Export `isvalid` and `mask`.
```
Comment (rplopes) [inline src/index.js]: Change to "Exports", like how we do on other projects? To avoid going out of sync in the future.

**Review by rplopes — CHANGES_REQUESTED** [review]
The build is failing because it's using the docker image `seegno/node:4-test`, which is using node v4, below the minimum supported version.

We need to update the CI config to target newer versions. We can probably simplify it as well and avoid having to specify docker images at all, like on other projects.

**Comment by cristianooliveira** [comment]
> The build is failing because it's using the docker image `seegno/node:4-test`, which is using node v4, below the minimum supported version.
> 
> We need to update the CI config to target newer versions. We can probably simplify it as well and avoid having to specify docker images at all, like on other projects.

Agree, i think it's best to do that.

**Review by rplopes — APPROVED** [review]
LGTM

**Pull request merged**